src/audio/dai-zephyr.c

```diff
@@ -1504,6 +1535,15 @@ int dai_zephyr_copy(struct dai_data *dd, struct comp_dev *dev, pcm_converter_fun
 
 	buffer_release(buf_c);
 
+	/* handle module runtime unbind */
+	if (!dd->local_buffer) {
+		set_new_local_buffer(dd, dev);
+
+		if (!dd->local_buffer) {
+			comp_warn(dev, "dai_zephyr_copy(): local buffer unbound, cannot copy");
```

Comment:
unbound --> unbind

Reply:
@btian1 I think this is a common "short hand" for a passive voice construct, the full version would be "buffer is unbound" or "buffer was unbound" or similar or maybe it should be interpreted as "buffer departed" but in any case "unbind" would be even more wrong

Reply:
hmm, native English speakers can correct, but I think unbound is correct. it's here as adjective (unbound buffer), but could be indeed read as shorthand for past tense verb ("buffer freed", "buffer allocated").